TS.fsx

```diff
@@ -531,8 +531,25 @@ let EmitDictionaries flavor =
         | "Object" -> Pt.printl "interface %s {" dict.Name
         | _ -> Pt.printl "interface %s extends %s {" dict.Name dict.Extends
 
+        let emitJsonProperty (p: ItemsType.Root) =
+            Pt.printl "%s: %s;" p.Name.Value p.Type.Value
+
+        let removedPropNames = 
+            getRemovedItems ItemKind.Property flavor 
+            |> Array.filter (matchInterface dict.Name)
+            |> Array.map (fun rp -> rp.Name.Value)
```

Comment:
`Set.contains` doesn't do any implicit conversions. You'd pay the construction cost once when building removedPropNames by adding `|> Set.ofArray`. 

*Generally*, Set.contains is O(lg n) or O(1) and Array.contains is O(n), so if removedPropNames is ever long-ish (10-100 or more items probably?) it will be worth it.

However, I can't find any documentation on `Array.contains`! Where does `Array.contains` come from? I see that Shared.fs adds `Seq.contains x = Seq.exists ((=)x)`, which is O(n), but I can't find a definition of `Array.contains` in our own code.